test/Integration/hoist-vector-transfer-brgemm.mlir

```diff
@@ -0,0 +1,59 @@
+// RUN: tpp-opt %s  | tpp-run -e entry --entry-point-result=void  -print > %t.1
+// RUN: tpp-opt %s --loop-invariant-code-motion  --vectorization-pass --loop-invariant-code-motion --hoist-vector-transfer | tpp-run -e entry --entry-point-result=void  -print > %t.2
+// RUN: diff %t.1 %t.2
```

Comment:
Make sure you remove `%t.1` and `%t.2`

Reply:
Thanks. Added command to remove `%t.1` and `%t.2`.